CMakeLists.txt

```diff
@@ -101,6 +101,11 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Os -Wno-sized-deallocation -fno-sized-d
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DDEBUG -DSANITIZE_PTRS")
 add_link_options(--sysroot ${CMAKE_BINARY_DIR}/Root)
 
+set(CMAKE_C_IMPLICIT_LINK_LIBRARIES "")
+set(CMAKE_CXX_IMPLICIT_LINK_LIBRARIES "")
+
+set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -nodefaultlibs -L${CMAKE_BINARY_DIR}/Root/usr/lib -lc -lm -lgcc")
```

Comment:
I'd really like to not set nodefaultlibs, because 1) it's a hack and 2) I'll need to set the same for SHARED linker flags? It also makes it really hard to rebuild libc and libm... 

The google searches i found for clearing the implicit link libraries seem to suggest it should make the linker not look for -lstdc++, but from experimenting only setting nodefaultlibs seems to have caused that....